pyvista/utilities/reader.py

```diff
@@ -625,6 +625,48 @@ def set_active_time_value(self, time_value):  # noqa: D102
     def set_active_time_point(self, time_point):  # noqa: D102
         self.reader.SetTimeValue(self.time_point_value(time_point))
 
+    @property
+    def cell_to_point_creation(self):
+        if self.reader.GetCreateCellToPoint():
+            return True
+        return False
+
+    @cell_to_point_creation.setter
+    def cell_to_point_creation(self, value):
+        if value:
+            self.reader.CreateCellToPointOn()
```

Comment:
The default in vtk is "on", but this is counterintuitive IMO, and unneccessary.  One can use a filter to translate cell data to point data later on.  I always turn this "off", so we could consider making the default "off" when instantiating the reader.

Reply:
I think that this makes sense to turn "off" as default and implemented in https://github.com/pyvista/pyvista/pull/1896/commits/0b52522e6e57fa180f9c33a753d50b7d4d88d3ac.  Users that need point data can use the `cell_data_to_point_data` filter, which should be identical to this option.

Reply:
After further investigation, the point data obtained using `reader.cell_to_point_creation=True` and `mesh.cell_data_to_point_data()` are different.  Since the point data is so useful, I think it is best to follow the vtk default here, but the example showcases this since it is confusing.